### PR TITLE
edits to notebooks 4 and 7

### PR DESCRIPTION
**Pull request by jon-holt**
- Edits to notebooks 4 and 7

### CODE REVIEW

File: content/Main_Notebooks/4 - Simple Filtering.ipynb [hunk 1]
```diff
@@ -398,25 +3027,206 @@
    "id": "b4706e46",
    "metadata": {},
    "source": [
-    "There are other ways to connect expressions. & handles \"and\", meaning both parts have to be true. | (which you can generally type as Shift+\\) means \"or\", meaning at least one of the parts has to be true.\n",
+    "There are other ways to connect expressions. `&` handles \"and\", meaning both parts have to be true. `|` (which you can generally type as Shift+\\) means \"or\", meaning at least one of the parts has to be true.\n",
```
Comment (michael-long88) [inline content/Main_Notebooks/4 - Simple Filtering.ipynb]: Not sure why, but it looks like the first slash in "(which you can generally type as Shift+\\)" got dropped when looking at in the browser, so it just looks like "(which you can generally type as Shift+)". Wrapping shift and \ in ` seems to fix the issue if both slashes are still showing up for you locally.

**Review by michael-long88 — CHANGES_REQUESTED** [review]
Everything is running that should be running and the appropriate cells are throwing errors. I just left one comment, but other than that, it looks good!

File: content/Main_Notebooks/4 - Simple Filtering.ipynb (hunk 1 above)
Comment (jon-holt) [reply content/Main_Notebooks/4 - Simple Filtering.ipynb]: You're right, it looks like the extra "\" got dropped. I just added it back in. See if this looks better.

**Review by michael-long88 — APPROVED** [review]
Looks good now!

**Pull request merged**